### DISCUSSION TITLE
Option to integrate with Open source LLMs

### DISCUSSION THREAD

**Discussion by nirajkmr007** [plandex-ai/plandex#86] [General]
Great Project! Excited to try out more.

Do you have plans to integrate this with open Source LLMs, giving more options than just openai APIs?

**Comment by danenania** [comment]
Thanks @nirajkmr007! Yes, I'm looking at adding some models from [together.ai](https://together.ai), as well as more flexibility to plug in any models that match the OpenAI api spec, very soon.

**Comment by danenania** [comment]
@nirajkmr007 This has now been released! You can use tons of open source models now via together.ai or openrouter.ai, among others--any provider that is OpenAI-compatible can be used. Check the release notes here for more details: https://github.com/plandex-ai/plandex/releases/tag/cli%2Fv0.9.0

**Reply by nasarshakil** [reply]
Hi danenania great job!
i am trying to get pdx to work with ollama but even though I successfully defined the model with a base url of http://localhost:11434/v1 I cannot 'tell' as it always comes back with a connection refused? Any ideas?

**Comment by SkybuckFlying** [comment]
I think /v1 is not longer there, try removing it, anyway I am also trying to use ollama with plandex... ollama can be reached from wsl/linux with windows firewall rule and 192.x.x.x ip etc.
